client/src/components/TranslationsPage/DonateModal/index.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import './style.css';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import Button from '../../common/Button';
+
+class DonateModal extends Component {
+  state = {
+    text: true,
+    audio: false,
+    video: false,
+  };
+
+  switchTab = (e) => {
+    const { target } = e;
+    const { textContent } = target;
```

Comment:
you can destructure the textContent attribute directly instead of doing it over two steps.
```suggestion
    const { textContent } = e.target;
```

Reply:
in the screenshot, we should give the tabs some padding or margin from the left corner of the modal.